src/bundle.c

```diff
@@ -682,15 +684,20 @@ gboolean check_bundle(const gchar *bundlename, RaucBundle **bundle, gboolean ver
 	if (r_context()->config->store_path) {
 		ibundle->storepath = r_context()->config->store_path;
 	} else {
-		gchar *strprfx;
+		gchar *path;
```

Comment:
What about simplifying lines 687-692 via the elvis operator: `gchar *path = ibundle->origpath ?: ibundle->path;`?

Reply:
A little less conversation, a little more action -> fixed it.